libs/adapters/src/utils/setupErrorHandlers.ts

```diff
@@ -46,14 +46,22 @@ export const setupErrorHandlers = (app: Express) => {
         error: error.message,
       });
     } else {
-      log.error(error.message, error, reqContext);
-      res.status(500);
-      res.json({
-        status: error.status,
-        error:
-          error.message ||
-          'Server error, unknown error thrown. Please try again later.',
-      });
+      if (error?.status < 500) {
+        log.warn(error.message || '', error, reqContext);
+        res.status(error.status);
+        res.json({
+          error,
```

Comment:
Any reason we're not providing the status here as part of res.json's argument like we do below?

Reply:
Added it. Technically unnecessary to add the status to the body especially when we already know that `error.status` is defined but I'm not sure if we have any client side error handling that relies on it so adding it to be safe.